src/application/app.rs

```diff
@@ -4,13 +4,21 @@ use crate::{application::State, backend::db};
 pub enum InputMode {
     Command,
     AddTask,
+    Prompt,
+}
+
+#[derive(PartialEq)]
+pub enum Actions {
```

Comment:
Maybe change name Actions to Action, because it isn't supposed to be a collective but enum

Reply:
done